lecture/forms.py

```
@@ -1,15 +1,41 @@
 from django import forms
-from .models import Lecture, LecturePDF
+from .models import Lecture, LecturePDF,LectureVideo
+from django.utils.text import slugify
+from courses.models import Module
+import os
 
 class LectureForm(forms.ModelForm):
     class Meta:
         model = Lecture
-        fields = ['title', 'classroom']
+        fields = ['module','title','description','video_url','content','order']
```

Ruff (0.8.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)